index.html

```diff
@@ -509,6 +448,129 @@ <h2>Subroutine: Supported display surface type</h2>
+                attribute has the value {{InvalidStateError}}.
+              </li>
+              <li>
+                If |currentEvent|.{{Event/type}} is not in
+                <a>permitted event types for zoom-setting</a>, return a promise [=reject|rejected=]
+                with a {{DOMException}} object whose {{DOMException/name}} attribute has the value
+                {{InvalidStateError}}.
+                <div class="note">
+                  <p>
+                    It follows from these steps that {{CaptureController/increaseZoomLevel()}},
+                    {{CaptureController/decreaseZoomLevel()}} and
+                    {{CaptureController/resetZoomLevel()}} are only callable with [=transient
+                    activation=], because [=permitted event types for zoom-setting=] only contains
+                    <a data-cite="DOM#dom-event-type">event types</a> that confer this activation.
+                  </p>
+                  <p>
+                    In fact, our API shape implies a stronger guarantee - whereas [=transient
+                    activation=] persists for several seconds after the user action, the API shape
+                    here limits zoom-setting to immediately after the user's action.
+                  </p>
+                </div>
+              </li>
+            </ol>
+          </li>
+          <li>
+            Let |currentZoomLevel| be [=this=].<a data-cite="!screen-capture/#dfn-source"
+              >[[\Source]]</a
+            >'s [=zoom level=]
+          </li>
+          <li>Let |targetZoomLevel| be 100.</li>
+          <li>
+            <p>If |zoomAction| is <code>-1</code> then:</p>
+            <ol>
+              <li>
+                If |currentZoomLevel| is the minimum value in [=supported zoom levels=], return a
+                promise [=reject|rejected=] with a {{DOMException}} object whose
+                {{DOMException/name}} attribute has the value {{InvalidStateError}}.
+              </li>
+              <li>
+                Otherwise, set |targetZoomLevel| to the value in [=supported zoom levels=] that
+                appears immediately after |currentZoomLevel|.
+              </li>
+            </ol>
+          </li>
+          <li>
+            <p>If |zoomAction| is <code>1</code> then:</p>
+            <ol>
+              <li>
+                If |currentZoomLevel| is the maximum value in [=supported zoom levels=], return a
+                promise [=reject|rejected=] with a {{DOMException}} object whose
+                {{DOMException/name}} attribute has the value {{InvalidStateError}}.
+              </li>
+              <li>
+                Otherwise, set |targetZoomLevel| to the value in [=supported zoom levels=] that
+                appears immediately after |currentZoomLevel|.
+              </li>
+            </ol>
+          </li>
+          <li>Let |P| be a new {{Promise}}.</li>
```

Comment:
```suggestion
          <li>Let |promise| be a new {{Promise}}.</li>
```

Reply:
Would prefer to discuss and potentially address this consistently throughout this document, rather than only in places which were touched by this particular PR.